ibis/expr/api.py

```diff
@@ -4418,11 +4418,32 @@ def _table_view(self):
     return new_view.to_expr()
 
 
-def _table_drop(self, fields):
+def _table_drop(self: ir.TableExpr,
```

Comment:
```suggestion
def _table_drop(self,
```
The convention with mypy is to not annotate `self`.

Reply:
Do you also type the return value.

Reply:
Yes. The idea is that mypy already knows which is the type of self, since it's the Python interpreter who provides it. But for the return, is the user (author of the function) who decides what is being returned, so mypy needs to be told.